arch/arm/src/Makefile

```diff
@@ -106,6 +107,8 @@ else
   SCRIPT_OPT  = --scatter=
 endif
 
+LIBRARY_OPT  ?= --library=
```

Comment:
So GCC linking with LD also use this option?

Reply:
Yes, or use -l by default, if armlink use --library?

Reply:
Done